test/electron/features/native-crash.feature

```diff
@@ -2,26 +2,29 @@ Feature: Native Errors
 
   Scenario: A minidump is uploaded on native error
     When I launch an app
+    And I wait for 1 session upload
     And I click "main-process-crash"
     Then I launch an app
+    And I wait for 2 session uploads
     Then the total requests received by the server matches:
       | minidumps | 1 |
+      | sessions  | 2 |
     And minidump request 0 contains a file form field named "upload_file_minidump"
     And minidump request 0 contains a form field named "event" matching "minidump-event.json"
 
   Scenario: Minidumps are retried when the network becomes available
     When I launch an app
+    And I wait for 1 session upload
     And I click "main-process-crash"
-    Given the server is unreachable
-    And I launch an app
-    Then the total requests received by the server matches:
-      | events    | 0 |
-      | minidumps | 0 |
-    When the server becomes reachable
+    Then the server is unreachable
+    And I launch an app with no network
+    Then the server becomes reachable
     And the app gains network connectivity
+    And I wait for 2 session uploads
```

Comment:
Is this wait necessary? "the total requests received by the server matches" will wait for 3 seconds already